common/model/bundle.h

```diff
@@ -96,6 +96,13 @@ static inline size_t bundle_transactions_size(bundle_transactions_t const *const
   return utarray_len(bundle);
 }
 
+/**
+ * @brief Re-initialize the bundle_transactions_t object.
+ *
+ * @param[in] bundle The pointer of bundle object.
+ */
+static inline void bundle_transactions_renew(bundle_transactions_t *bundle) { utarray_clear(bundle); }
```

Comment:
Please rephrase the doc as well.

```suggestion
static inline void bundle_transactions_clear(bundle_transactions_t *bundle) { utarray_clear(bundle); }
```